internal/repository/index_test.go

```diff
@@ -329,7 +330,20 @@ func TestIndexUnserialize(t *testing.T) {
 	}
 }
 
+var (
+	benchmarkIndexJSON     []byte
+	benchmarkIndexJSONOnce sync.Once
+)
+
+func initBenchmarkIndexJSON() {
+	idx, _ := createRandomIndex(rand.New(rand.NewSource(0)))
+	var buf bytes.Buffer
+	idx.Encode(&buf)
+	benchmarkIndexJSON = buf.Bytes()
+}
+
 func BenchmarkDecodeIndex(b *testing.B) {
+	benchmarkIndexJSONOnce.Do(initBenchmarkIndexJSON)
 	b.ResetTimer()
 
 	for i := 0; i < b.N; i++ {
```

Comment:
s/docExample/benchmarkIndexJSON/
for the line below

Reply:
Done.